schema/base/land.yaml

```diff
@@ -0,0 +1,74 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Schema
+description: Land
```

Comment:
I wonder if there would be added value in copy-pasting texts from docusaurus/docs/themes/base/index.mdx#Current Types to land.yaml, water.yaml and landUse.yaml?

Reply:
Good call, done.